server/libs/database/src/entities/shared-link.entity.ts

```diff
@@ -0,0 +1,51 @@
+import { Column, Entity, Index, ManyToMany, ManyToOne, PrimaryColumn, Unique } from 'typeorm';
+import { AlbumEntity } from './album.entity';
+import { AssetEntity } from './asset.entity';
+
+@Entity('shared_links')
+@Unique('UQ_sharedlink_key', ['key'])
+export class SharedLinkEntity {
+  @Index('IDX_sharedlink_id')
```

Comment:
If I'm not mistaken primary columns have indices by default/design, so this is not needed.